test/e2e/container_probe.go

```diff
@@ -43,7 +43,7 @@ var _ = Describe("Probing container", func() {
 		expectNoError(err)
 		startTime := time.Now()
 
-		Expect(wait.Poll(poll, 90*time.Second, func() (bool, error) {
+		Expect(wait.Poll(poll, 120*time.Second, func() (bool, error) {
```

Comment:
I actually think the second phase of the first test should have a shorter timeout. The readiness should be set pretty quickly after the initial delay is up, and if it doesn't it's likely a bug (e.g. https://github.com/kubernetes/kubernetes/issues/17656). I'd suggest 30-60s, and only increasing it if that doesn't seem to be enough.